scripts/get-language-providers.sh

```diff
@@ -28,7 +28,7 @@ download_release() {
 # * When updating .Net, you should also update PulumiDotnetSDKVersion in pulumi/pkg/codegen/testing/test/helpers.go
 #
 # shellcheck disable=SC2043
-for i in "github.com/pulumi/pulumi-java java v1.8.0" "github.com/pulumi/pulumi-yaml yaml v1.15.2" "github.com/pulumi/pulumi-dotnet dotnet v3.77.0"; do
+for i in "github.com/pulumi/pulumi-java java v1.9.0" "github.com/pulumi/pulumi-yaml yaml v1.16.0" "github.com/pulumi/pulumi-dotnet dotnet v3.78.0"; do
```

Comment:
You need to bump https://github.com/pulumi/pulumi/blob/3180549e6ed7d09d2c0c721c16b34fe55979cff8/pkg/codegen/testing/test/helpers.go#L381, see the comment above

Reply:
Ah yes of course!  Thanks for catching this!